src/app/cli/src/coda_run.ml

```diff
@@ -337,8 +337,22 @@ let run_snark_worker ?shutdown_on_disconnect:(s = true) ~client_port
       |> ignore
 
 let handle_crash e =
-  Core.eprintf
-    !{err|
+  match Monitor.extract_exn e with
+  | Coda_networking.No_initial_peers ->
+      Core.eprintf
+        !{err|
+
+  ☠  Coda Daemon failed to connect to any initial peers.
+
+  This may mean your daemon tried to join a network for a different chain.
```

Comment:
Suggest changing this to:

"You might be trying to connect to a different network, or need to troubleshoot your configuration. See https://codaprotocol.com/docs/troubleshooting/ for details."